### PR TITLE
Remove compile dep to servlet and jetty

### PR DESCRIPTION
**Pull request by drtechniko**
That caused dependency issues.

### CODE REVIEW

File: karyon2-admin/build.gradle [hunk 1]
```diff
@@ -18,8 +18,8 @@ apply plugin: 'osgi'
 dependencies {
     dependencies {
         compile 'javax.ws.rs:jsr311-api:1.1.1'
-        compile 'javax.servlet:servlet-api:2.5'
-        compile 'org.mortbay.jetty:jetty:6.1.26'
+        provided 'javax.servlet:javax.servlet-api:3.0.1'
+        provided 'org.mortbay.jetty:jetty:6.1.26'
```
Comment (brharrington) [inline karyon2-admin/build.gradle]: It seems odd to make jetty provided when it is being used as an embedded server. What if the application is running outside of a servlet container or using a container other than jetty such as tomcat? Is the user expected to supply and keep track of the appropriate version of jetty?

Comment (elandau) [inline karyon2-admin/build.gradle]: What was the specific issue that prompted this PR?   Also, this is a very old version of jetty.  Perhaps we should consider changing this to the latest version of jetty in the org.eclipse.jetty org.

Comment (brharrington) [inline karyon2-admin/build.gradle]: Note there were problems the last time it was updated to jetty 9 (see #220). So it would need some testing.

**Comment by drtechniko** [comment]
I'm just doing an upgrade of 3.0.1 for this one. We can investigate upgrading jetty later.

**Comment by noel-yap** [comment]
Just FYI, since the artifact coordinate changed and the package names haven't changed, both versions of the library can get brought in and wind up loading the wrong classes. I think creating a Gradle Resolution Rule (https://github.com/nebula-plugins/gradle-resolution-rules) can help mitigate issues assuming clients use the plugin.

**Comment by noel-yap** [comment]
:+1:

**Pull request merged**